server/controllers/recommendation/ppcs/indeterminateSentence/selectIndeterminatePpudSentenceController.ts

```diff
@@ -73,7 +74,9 @@ async function post(req: Request, res: Response, next: NextFunction) {
     token: res.locals.user.token,
   })
 
-  // TODO MRD-2687 redirect to the next page once it's ready
+  const { urlInfo } = res.locals
```

Comment:
This is the closest line to updateRecommendation but: do we have a plan for when we introduce saving a copy of the sentence details that will be edited. They are required for the edit pages that development has started on and are awaiting on.

Reply:
Resolving this conversation, as this has been discussed and agreed on in a call.